.gitignore

```diff
@@ -21,3 +21,5 @@ pom.xml
 /data/andb-*.json
 /data/img
 /.lein-failures
+
+/data/db
```

Comment:
This does not need to be committed / pulled I guess. I did not intend it to sneak in...